backend/src/test/java/com/woowacourse/levellog/presentation/TeamControllerTest.java

```diff
@@ -584,6 +612,124 @@ void teamNotFound_Exception() throws Exception {
         }
     }
 
+    @Nested
+    @DisplayName("closeInterview 메서드는")
+    class CloseInterview {
+
+        @Test
+        @DisplayName("존재하지 않는 팀의 인터뷰를 종료하려고 하면 예외가 발생한다.")
+        void close_notFoundTeam_exceptionThrown() throws Exception {
+            // given
+            given(jwtTokenProvider.getPayload(TOKEN)).willReturn("4");
+            given(jwtTokenProvider.validateToken(TOKEN)).willReturn(true);
```

Comment:
```suggestion
            mockLogin();
```
develop 리베이스 후 변경 해주세요. 😉